modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js

```diff
@@ -808,12 +814,27 @@ SideNavigation.prototype = {
 		}
 	},
 
+	_subscribeReducedMotion() {
+		const instance = this;
+
+		Liferay.Loader.require('frontend-js-web/index', ({isReducedMotion}) => {
```

Comment:
Apparently we can't import other modules from the portal to files that are being declared in the global, probably due to the implementation of the build that is done with webpack in that part so we have to declare it that way here, I realized that there is the same pattern for the declaration of `openModal ` and components related to Modal.